_component_design/switches.md

```diff
@@ -14,72 +14,91 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-<!--{% include todo.html content="Update focus states, expand on inline validation, and improve
-keyboard accessibility." %}-->
 
-The switch should only be used in **mobile** designs. Enabling preferences or
-settings is a prime use case for this component. The switch can be used with or
-without text, depending on the context provided in the switch label. If more
-complex text is needed in the switch than yes/no or on/off, use a **toggle**.
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-## Default
+### When to use
+{: .hxSubSectionTitle}
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/selectors-switch.png){:width="30%"}
+Use a switch if the user is turning something on or off, or saying yes or no to a question. There is no neutral option; the answer is either set to on or off. Enabling preferences or settings is a prime use case for this component. The switch should always be used with text. 
+
+If more complex text is needed in the switch than yes/no or on/off, use a toggle. A switch should NOT be used for things like user agreements or if there are more than two options available.
```

Comment:
Use a toggle if more complex text is required. * (just shorten the sentence)

Reply:
should not* (no need for caps) be used for more complex assignments, such as user agreements.*

Reply:
Shortened the first edit and opted for bold instead of uppercase. 

"Use a toggle if more complex text is required. A switch **should not** be used for things like user agreements or if there are more than two options available."